### ISSUE TITLE
Questions about the code

### ISSUE THREAD

**Issue by jzwhitemountain** [NahuelCostaCortez/Remaining-Useful-Life-Estimation-Variational#3]
Hello, can this code run directly to achieve the similar effect in the paper, or do I need to make any changes? I have tried for many days, but I can't do it. Is there a problem with my settings or the code environment？

This is how I operate：
python main.py
FD001

And then I got this result:
train set RMSE:23.477054595947266, R2:0.684546574723093
test set RMSE:21.99192231175095, R2:0.7199298389605739

**Comment by NahuelCostaCortez** [comment]
Hello jzwhitemountain,

Thank you very much for trying out the repo and reporting your issues! I looked into it and it seems that the problem lies in the LR. The optimizer is set to "adam" with its default parameters. This leads to a suboptimal training process. 

Until I update the repository with the best hyperparameters you can run:

https://github.com/NahuelCostaCortez/Remaining-Useful-Life-Estimation-Variational/blob/main/RULRVE.ipynb

Here the LR is initialized to 0.001, which leads to competitive results in just 10 epochs.

Let me know if you have further questions.

Best,
Nahuel

**Comment by jzwhitemountain** [comment]
Thank you for your prompt reply.
